parts/chapters/subsections/11.3/EXCEL.fodt

```diff
@@ -4100,7 +4100,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
      </table:table-row>
     </table:table>
     <text:h text:style-name="P18345" text:outline-level="4" text:is-list-header="true"><text:bookmark-start text:name="__RefHeading___Toc10367_370116838819131"/>Description<text:bookmark-end text:name="__RefHeading___Toc10367_370116838819131"/></text:h>
-    <text:p text:style-name="_40_TextBody">This keyword activates the writing out of the RSM file data in a format that can easily be loaded into Microsoft&apos;s EXCEL spreadsheet program or LibreOffice’s CALC spreadsheet program. The RSM file output is activated by the RUNSUM keyword in the SUMMARY section.</text:p>
+    <text:p text:style-name="_40_TextBody">This keyword activates the writing out of the RSM file data in a format that can easily be loaded into Microsoft&apos;s <text:a xlink:href="#__RefHeading___Toc210148_2884651453">EXCEL</text:a> spreadsheet program or LibreOffice’s CALC spreadsheet program. The RSM file output is activated by the <text:a xlink:href="#__RefHeading___Toc210156_2884651453">RUNSUM</text:a> keyword in the <text:a xlink:href="#__RefHeading___Toc43949_784232322">SUMMARY</text:a> section.</text:p>
```

Comment:
EXCEL might be a keyword, but here it a brand of a software provider. We should probably change the case.